aries-site/src/examples/components/toast-notifications/ToastNotificationExample.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { Box, Button, Text, Diagram, Stack } from 'grommet';
+import { Notification } from './ToastNotification';
+
+export const ToastNotificationExample = () => {
+
+    const [visible, setVisible] = useState();
+
+    const onOpen = () => {
+        setVisible(true);
+        setTimeout(() => {
+        setVisible(undefined);
+        }, 8000);
+    };
+
+    const onClose = () => setVisible(undefined);
+
+    return (
+        <>
+            <Box align="center" gap="medium">
+                <Text textAlign="center">
+                    For example purposes, this Toast Notification
+                    will disappear after 8 seconds. This is the 
+                    recommended lifetime for a toast, as it allows
+                    screen readers to relay the content in a 
+                    notification. 
+                </Text>
+                <Button 
+                    label="Show me the Notification" 
+                    onClick={onOpen} 
+                    primary 
+                />
+            </Box>
+            <Box align="center" gap="small">
+                {visible && <Notification 
+                    toast
+                    status="good"
+                    title="Toast success!"
+                    message="The operation was successful"
+                    onClose={onClose}
+                />}
+            </Box>
+        </>
+    );
+};
+
+const color = 'black';
+const anchor = 'vertical';
+const thickness = 'xxxsmall';
+const type = 'rectilinear';
+
+const connections = [
+    {
+        anchor,
+        type,
+        color,
+        thickness,
+        fromTarget: '1',
+        toTarget: 'status-indicator',
+    },
+    {
+        anchor,
+        type,
+        color,
+        thickness,
+        fromTarget: '2',
+        toTarget: 'content',
+    },
+    {
+        anchor,
+        type,
+        color,
+        thickness,
+        fromTarget: '3',
+        toTarget: 'close-button',
+    },
+];
+
+export const ToastDiagram = () => (
```

Comment:
Style/convention comment. Move to its own contained file and leave ToastNotificationExample only showing the example?

Also, the intent of the examples are for developers to be able to copy/paste the code example into their project. The toast diagram piece here would be extraneous, correct?

Reply:
makes sense to me!